docs/changes.xml

```diff
@@ -43,6 +43,13 @@ prefer system crypto policy, instead of hardcoding a default.
 </para>
 </change>
 
+<change type="feature">
+<para>
+new configuration option 'max_headers_size' for 'settings.http' to set the
+maximum allowed size of the HTTP headers (incl URL). Defaults to 8192 bytes.
```

Comment:
`s/incl URL/including request line` would be more accurate I think

Reply:
The 'technical' term seems to be [_start-line_](https://www.rfc-editor.org/rfc/rfc2616#section-4)